briefings.html.haml

```diff
@@ -111,6 +111,15 @@ title: Events & Briefings
           %p Catch up on anything you might have missed.
         .container.wow.fadeInUp
           .col-sm-6.col-md-12
+             .service-02
```

Comment:
This `.service-02` tag has one too many spaces in front of it; I am not sure it will build correctly.